src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -5501,5 +5496,203 @@ static void Main()
                 );
             var compVerifier = CompileAndVerify(compilation, expectedOutput: expectedOutput);
         }
+
+        [Fact]
+        public void IsNot_01()
```

Comment:
>IsNot_01 [](start = 20, length = 8)

Consider adding a test that is not an `is`, but a switch expression arm. Or do we already have one?

Reply:
We already have many. There is no change of semantics for those.

---
In reply to: [430771480](https://github.com/dotnet/roslyn/pull/44513#discussion_r430771480) [](ancestors = 430771480)